e2e/__tests__/stackTraceSourceMapsWithCoverage.test.ts

```diff
@@ -15,9 +15,5 @@ it('processes stack traces and code frames with source maps with coverage', () =
   );
   run('yarn', dir);
   const {stderr} = runJest(dir, ['--no-cache', '--coverage']);
-
-  // Should report an error at source line 13 in lib.ts at line 10 of the test
```

Comment:
can keep the comment probably, to explain what we're checking 🙂

Reply:
sure, i've put this back.